featuretools/synthesis/dfs.py

```diff
@@ -198,10 +201,11 @@ def dfs(entities=None,
 
             Valid keyword arguments for LocalCluster will also be accepted.
 
-        return_variable_types (list[Variable] or str, optional): Types of
-                variables to return. If None, default to
-                Numeric, Discrete, and Boolean. If given as
-                the string 'all', use all available variable types.
+        return_types (list[woodwork.ColumnSchema] or str, optional):
+            List of ColumnSchemas defining the types of
+            columns to return. If None, defaults to returning all
+            numeric, categorical and boolean types. If given as
+            the string 'all', use all available variable types.
```

Comment:
instead of "use all available variable types", perhaps "returns all available types"